### PR TITLE
Add toBeEnabled matcher

### LINKED ISSUE #76: New toBeEnabled() matcher
### Describe the feature you'd like:

I'd like a matcher to check if a specific input element is enabled.

### Describe alternatives you've considered:

Today, I check for the input being enable using the `not.toBeDisabled()` matcher. I think It would be better for the read/clarity of the test, I be able to write `.toBeEnabled()`.

### PR DESCRIPTION
**Pull request by lourenci**
**What**:

Add `toBeEnabled` matcher to the library. This PR closes #76.

**Why**:

It feels more natural to write the specs. Most of us don't like to double negation tests.

**How**:

I just "negated" the `toBeDisabled` matcher, including the spec file. To aim that, I extracted the logic to the utils file.

I don't know if that is the better approach to aim that, I don't even know if the `utils.js` is the better file to contain some "shared" logic to be shared between matcher files. So, it would be good that someone take some time to review that.

**Checklist**:

* [X] Documentation
* [X] Tests
* [ ] Updated Type Definitions
* [X] Ready to be merged 
* [X] Added myself to contributors table

### CODE REVIEW

**Comment by lourenci** [comment]
I don't have any arguments to keep what I've done. The `toBeEnabled` matcher is just a negated form to `toBeDisabled`. I'll do that. :)

**Comment by lourenci** [comment]
@gnapse it's done. :)

File: README.md [hunk 1]
```diff
@@ -733,10 +746,10 @@ Thanks goes to these people ([emoji key][emojis]):
...
...<br /><sub><b>Iwona</b></sub>](https://github.com/callada)<br />[💻](https://github.com/gnapse/jest-dom/commits?author=callada "Code") [📖](https://github.com/gnapse/jest-dom/commits?author=callada "Documentation") [⚠️](https://github.com/gnapse/jest-dom/commits?author=callada "Tests") | [<img src="https://avatars0.githubusercontent.com/u/840609?v=4" width="100px;"/><br /><sub><b>Lewis</b></sub>](https://github.com/6ewis)<br />[💻](https://github.com/gnapse/jest-dom/commits?author=6ewis "Code") | [<img src="https://avatars3.githubusercontent.com/u/2339362?v=4" width="100px;"/><br /><sub><b>Leandro Lourenci</b></sub>](https://blog.lourenci.com/)<br />[🐛](https://github.com/gnapse/jest-dom/issues?q=author%3Alourenci "Bug reports") [📖](https://github.com/gnapse/jest-dom/commits?author=lourenci "Documentation") [💻](https://github.com/gnapse/jest-dom/commits?author=lourenci "Code") [⚠️](https://github.com/gnapse/jest-dom/commits?author=lourenci "Tests") | [<img src="https://avatars1.githubusercontent.com/u/626420?v=4" width="100px;"/><br /><sub><b>Shukhrat Mukimov</b></sub>](https://github.com/mufasa71)<br />[🐛](https://github.com/gnapse/jest-dom/issues?q=author%3Amufasa71 "Bug reports") |
```
Comment (gnapse) [inline README.md]: Can you check why your PR has changed so much of this? 'm not sure it should't be this much.

Comment (lourenci) [reply README.md]: It looks right. I checked out the `.all-contributorsrc` and the `README.md` to the master version and regenerated it again, there is no difference between this version and the new one.

**Pull request merged**